static/js/containers/admin/EditChannelModeratorsPage.js

```diff
@@ -35,7 +42,37 @@ export class EditChannelModeratorsPage extends React.Component<Props> {
 const mapDispatchToProps = (dispatch: Dispatch<*>, ownProps) => ({
   loadMembers: () =>
     dispatch(actions.channelModerators.get(getChannelName(ownProps))),
-  loadChannel: () => dispatch(actions.channels.get(getChannelName(ownProps)))
+  loadChannel:  () => dispatch(actions.channels.get(getChannelName(ownProps))),
+  formValidate: (payload: FormValue<AddMemberForm>) =>
```

Comment:
This is duplicating a lot of what `withForm` and friends give you for free, is there a reason to avoid using that here?

Reply:
What do you mean? Can you go into more detail?

Reply:
Combining `withForm` (https://github.com/mitodl/open-discussions/blob/master/static/js/hoc/withForm.js) and `configureForm` (https://github.com/mitodl/open-discussions/blob/master/static/js/lib/forms.js#L36) allows you to get rid of a lot of the boilerplate code like this around forms.

Take a look at this page as an example: https://github.com/mitodl/open-discussions/blob/master/static/js/containers/auth/LoginPage.js